### PR TITLE
Add tokens list

### PR DESCRIPTION
**Pull request by neuodev**
tokenlist

@Marta-Ryzhok  Feel free to request changes

### CODE REVIEW

**Comment by Marta-Ryzhok** [comment]
@AhmedIbrahim336 
if there is 50 type of tokens it will be listed below the input?

**Comment by neuodev** [comment]
> if there is 50 type of tokens it will be listed below the input?
@Marta-Ryzhok Feel free to suggest another option

File: packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js [hunk 1]
```diff
@@ -403,7 +408,13 @@ export default class WalletSendForm extends Component<Props> {
         })
       ]
     })()
-
+    const tokenListClasses = classnames([
+      styles.tokenList,
+      {
+        [styles.show]: this.props.shouldSendAll && 
+           this.form.$('selectedAmount').value === tokenId
```
Comment (yushih) [inline packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js]: Should this be `this.form.$('selectedToken').value === tokenId` ?

File: packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js [hunk 2]
```diff
@@ -510,6 +524,15 @@ export default class WalletSendForm extends Component<Props> {
             )}
           />
 
+          <div className={tokenListClasses}>
+            <h1>{intl.formatMessage(messages.willSendAll)}</h1>
+            { tokenOptions.map(token => (
+              <p key={token.id} className= "accordion-item__paragraph" >
```
Comment (yushih) [inline packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js]: What is the class `accordion-item__paragraph` for? Seems there's no style that applies to this class.

Also please fix the indentation.

**Review by yushih — CHANGES_REQUESTED** [review]
I think there's a problem: if I select a token and select "send all", then I'm only sending this token so it should not show the token list, but it does:
tokenlist

**Comment by Marta-Ryzhok** [comment]
> I think there's a problem: if I select a token and select "send all", then I'm only sending this token so it should not show the token list, but it does:
> tokenlist

Yeah you’re right, shouldn’t be a token list below

File: packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js (hunk 2 above)
Comment (neuodev) [reply packages/yoroi-extension/app/components/wallet/send/WalletSendForm.js]: I replaced it 👍

**Review by yushih — CHANGES_REQUESTED** [review]
In the last commit, 3 irrelevant `package-lock.json` files are checked in. Please remove the changes. Otherwise LGTM.

**Comment by yushih** [comment]
@AhmedIbrahim336, please resolve the conflict.

**Review by yushih — CHANGES_REQUESTED** [review]
@AhmedIbrahim336 , please resolve the conflict. And be careful not to break the behavior of the "send all" dropdown.

**Comment by neuodev** [comment]
@yushih  Done

**Review by yushih — APPROVED** [review]

**Pull request merged**